misc/authorize.freeradius

```diff
@@ -0,0 +1,221 @@
+#
```

Comment:
move all misc/ files into misc/freeradius/  -- e.g. so instead of this file being misc/authorize.freeradius it would be misc/freeradius/authorize